packages/core/test/utils.spec.js

```diff
@@ -0,0 +1,309 @@
+const mockArgv = require('mock-argv');
+const mock = require('mock-fs');
+
+const utils = require('../src/utils');
+
+describe('Utils', () => {
+    describe('.lang()', () => {
+        it('returns correct info for scss', () => {
+            const lang = utils.lang('file.scss');
+            expect(lang.name).toEqual('SCSS');
+            expect(lang.mode).toEqual('scss');
+            expect(lang.scope).toEqual('source.scss');
+        });
+
+        it('returns correct info for nunjucks variants', () => {
+            const expected = {
+                name: 'HTML+Django',
+                mode: 'django',
+                scope: 'text.html.django',
+                color: null,
+            };
+            expect(utils.lang('file.nunjucks')).toEqual(expected);
+            expect(utils.lang('file.nunjs')).toEqual(expected);
+            expect(utils.lang('file.nunj')).toEqual(expected);
+            expect(utils.lang('file.nj')).toEqual(expected);
+            expect(utils.lang('file.jinja2')).toEqual(expected);
+            expect(utils.lang('file.j2')).toEqual(expected);
```

Comment:
The [official extension](https://mozilla.github.io/nunjucks/templating.html) is `njk`, you skipped it because you expect `fang` to cover that already?

Reply:
yup, we don't have a switch case in our code for that extension.